bindings/pyroot/pythonizations/test/numbadeclare.py

```diff
@@ -77,7 +77,7 @@ def fn1(x):
 
         self.assertTrue(hasattr(fn1, "__cpp_wrapper__"))
         self.assertTrue(type(fn1.__cpp_wrapper__) == str)
-        self.assertEqual(sys.getrefcount(fn1.__cpp_wrapper__), 2)
+        self.assertEqual(sys.getrefcount(fn1.__cpp_wrapper__), 3)
```

Comment:
Why are we even testing this? I believe it's not necessary and as your commit message rightly says, fragile. Let's just remove it

Reply:
I'm not sure, I'm not familiar with the tests.

Right now things are quite stable with only this little change of the reference value, but if you agree that it can be removed I'm happy to do this as soon as the test fails somewhere again!